CHANGELOG.md

```diff
@@ -5,9 +5,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 ## [Unreleased]
 ### Added
-- `crossbeam-channel` dependency was bumped to 0.5
 
 ### Changed
+- `crossbeam-channel` dependency was bumped to 0.5
+- `nom` dependency was bumped to 6.0.1
+- `base64` dependency was bumped to 0.13
```

Comment:
should probably mention a bump in minimum rust version here (which I'm 100% on board with fwiw)